src/renderer/inputSystem/index.ts

```diff
@@ -52,6 +52,16 @@ const defaultActions: Action[] = [
       })
     },
   },
+
+  {
+    name: 'wristlights',
+    bindings: [{ type: 'gamepadBtnDown', button: buttonMappings.X }],
+    perform: () => {
+      rosClient
+        .callService({ name: '/capra/wrist_light_toggle' })
```

Comment:
not a big fan of spliting a line like that, but if it makes software people happy and still works, why not

Reply:
That's style rules we have on the UI :)